bin/__tests__/changeAvailability.test.js

```diff
@@ -65,6 +65,17 @@ describe('changeAvailability', () => {
       false
     );
 
+    expect(mockInquirer.prompt).toHaveBeenCalledWith([
+      {
+        type: 'confirm',
+        name: 'confirmPackageRelease',
+        message:
+          'An extension package with the name extension_name and development availability ' +
+          'was found on the server and will be released. The extension package version is ' +
+          '1.0.0. Is this the extension package you would like to release to private ' +
+          'availability?'
+      }
```

Comment:
This might be even better, but whatever you think:

> An extension package with the name extension_name at version 1.0.0 with development availability was found on the server. Would you like to release this extension package to private availability?

I also find it nice to bold the important parts [like we do in the uploader](https://github.com/Adobe-Marketing-Cloud/reactor-uploader/blob/bb8e944544678e47677e872ace1d435764a58829/bin/uploadZip.js#L33).

Reply:
Chalk doesn't work with inquirer.

Reply:
Changed the message.